airbyte-integrations/connectors/source-oracle/src/test-integration/java/io/airbyte/integrations/source/oracle/OracleSourceDatatypeTest.java

```diff
@@ -146,7 +147,8 @@ protected void initTests() {
             .sourceType("NUMBER")
             .airbyteType(JsonSchemaType.NUMBER)
             .addInsertValues("null", "1", "123.45", "power(10, -130)", "9.99999999999999999999 * power(10, 125)")
-            .addExpectedValues(null, "1", "123.45", String.valueOf(Math.pow(10, -130)), String.valueOf(9.99999999999999999999 * Math.pow(10, 125)))
+            .addExpectedValues(null, "1", "123.45", String.valueOf(Math.pow(10, -130)),
+                "999999999999999999999000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000")
```

Comment:
Is this not possible to have the last expected value be 
```java
String.valueOf(9.99999999999999999999 * Math.pow(10, 125)
```

This would be difficult to understand, but if the usage of `Math.pow(10,125)` is not an option can we add a comment to this test case explaining how come this string is the way it is?

Reply:
The `String.valueOf` we used to have was printed as "1.0E126" - the scientific notation.
I couldn't find a way to compel BigDecimal to accept a value using `pow` that is not printed as a `1` with 126 `0`s, so had to resort to direct string. 
I'll add a comment.

Reply:
I might add in the comment the reason how come it's a plain string representation, like something along the lines of 
```java
"because normalization expects values in integer strings whereas `Math.pow(10, 125)` returns a scientific notation
```

Reply:
Added. thank you